pkgs/firedragon/src.json

```
@@ -0,0 +1,15 @@
+{
+  "packageVersion": "111.0.1-1",
+  "source": {
+    "rev": "111.0.1-1",
+    "sha256": "045nisl000ll0pzir9zhrkbbkl87bsd38mygx7gz9kv6p0pppl7i"
```

```suggestion
    "hash": "sha256-8dB7L7hmz/Tf6c9XNJpeB9G51szwpxz/BZQCAKiOthA="
```

This came from: `nix hash to-sri 'sha256:045nisl000ll0pzir9zhrkbbkl87bsd38mygx7gz9kv6p0pppl7i'`